Context.sublime-menu

```diff
@@ -29,7 +29,12 @@
             },
             {
                 "command": "lsp_code_actions",
-                "caption": "Apply Code Action…"
+                "caption": "Code Action…"
```

Comment:
Changed caption here as this command doesn't apply right away, it requires further input and then applies.

Reply:
It should also be changed in Main.sublime-mennu

Reply:
Never mind, it's not in the main menu